src/main/java/uk/gov/hmcts/darts/audio/exception/AudioApiError.java

```diff
@@ -84,7 +84,11 @@ public enum AudioApiError implements DartsApiError {
     REASON_IS_MARKED_FOR_DELETION(
         AddAudioErrorCode.MEDIA_HIDE_ACTION_REASON_MARKED_FOR_DELETION.getValue(),
         HttpStatus.NOT_IMPLEMENTED,
-        AddAudioErrorCode.MEDIA_HIDE_ACTION_REASON_MARKED_FOR_DELETION.getValue());
+        AddAudioErrorCode.MEDIA_HIDE_ACTION_REASON_MARKED_FOR_DELETION.getValue()),
+    ADMIN_SEARCH_CRITERIA_NOT_SUITABLE(
+        AddAudioErrorCode.ADMIN_SEARCH_CRITERIA_NOT_SUITABLE.getValue(),
+        HttpStatus.BAD_REQUEST,
+        AddAudioErrorCode.ADMIN_SEARCH_CRITERIA_NOT_SUITABLE.getValue());
```

Comment:
I think this should be using the `AddAudioTitleErrors` enum to get the meaningful title, rather than `AddAudioErrorCode` which only holds the error code.